HW_1/task_1.py

```diff
@@ -0,0 +1,12 @@
+
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+n = int(input('Введите трехзначное число: '))
+a = n % 10
+n = n // 10
+b = n % 10
+n = n // 10
```

Comment:
Решение может быть на одно выражение короче.
```suggestion
a = n % 10
b = n // 10 % 10
n = n // 100
```